examples/postman/setup_postman

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
I think having this file and the other `generate_postman_collection` file is confusing. When I first checked out the branch my first instinct was to run the version in `bin` which wouldn't setup the default variables. Why do we need both of them?

Reply:
I combined the two scripts into one. `bin/generate_postman_collection` will now:

- by default: generate collection, add scripts and default variables and empty auth variables
- with `--fill-env-vars` flag: generate collection, add scripts and variables filled with local dev env specific auth creds